### PR TITLE
venueless welcome page 

### PR DESCRIPTION
**Pull request by blulady**
This is the website page for the venueless welcome page to maintain one source of truth and maintain branding.
this shouldn't be merged until we can add links to the two tracks.

### CODE REVIEW

**Comment by blulady** [comment]
welcome_page1

welcome_page2

**Comment by blulady** [comment]
I liked the Sponsors & the latest news but I understand if we don't want these in Venueless.

File: src/venueless-welcome-page.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+title: Venueless Welcome Page
+description: Online Welcome
+---
+
+<div class="pt-8 pb-0 bg-repeat border-b-4 border-black lg:pt-2 bg-lines">
+  <div class="wrapper">
+    <div class="flex flex-col gap-8 lg:flex-row">
+      <section class="self-center w-full pb-4 lg:flex-1">
+        <h1 class="mb-4 text-4xl font-extrabold leading-tight lg:text-5xl">
+          Welcome to DjangoCon Online
```
Comment (PeterGrand) [inline src/venueless-welcome-page.html]:
```suggestion
          Welcome to DjangoCon US 2024 Online
```

**Comment by PeterGrand** [comment]
> I liked the Sponsors & the latest news but I understand if we don't want these in Venueless.

they look good!

**Review by PeterGrand — APPROVED** [review]

**Pull request merged**